snippets.yaml

```diff
@@ -0,0 +1,70 @@
+Dataframes.Table:
+  $global:
+    - name: Table.read "path/to/data"
+      documentation: Read table from file.
+  
+  Table:
+    - name: write "./myTable.csv"
+      documentation: Save the table as a file. Supported file formats are: csv, txt, xlsx and feather.
+    - name: describeNa
+      documentation: Calculate number and percentage of null values for each column.
+    - name: describe
+      documentation: Calculate statistics: standard deviation, minimum, maximum, mean, median quantile 0.25 and 0.75 for each column. 
```

Comment:
Not requesting a change, just wondering.  Perhaps we should omit listing the statistics? 

What is the preferred length for the description? This one feels a little long and I'm not sure to what degree we want to duplicate documentation.
Also, I'm concerned about the number of places in code that will become implicitly obsolote once we add/remove a calculated statistic.

Possible counterargument: searchabality.